src/google.rs

```diff
@@ -32,3 +54,44 @@ impl GoogleChatMessage {
         ))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_build_threaded_webhook_with_placeholder() {
+        let result = GoogleChatMessage::build_webhook_url(&String::from("https://example.com/ABCDEF?threadKey={threadKey}"), &String::from("1234"));
+        match result {
+            Ok(url) => assert_eq!(url, String::from("https://example.com/ABCDEF?messageReplyOption=REPLY_MESSAGE_FALLBACK_TO_NEW_THREAD&threadKey=1234")),
+            Err(_) => assert!(false),
+        }
```

Comment:
```suggestion
        assert_eq!(url, Ok(String::from("https://example.com/ABCDEF?messageReplyOption=REPLY_MESSAGE_FALLBACK_TO_NEW_THREAD&threadKey=1234")))
```

Is it possible to assert on a result perhaps?

Reply:
That isn't possible (Result doesn't support equality checks), but `Result::unwrap` will work well in tests.